### PR TITLE
Replace phantom.js usage with headless chrome

### PR DESCRIPTION
**Pull request by sivakumar-kailasam**
//cc @Turbo87

### CODE REVIEW

File: blueprints/app/files/testem.js [hunk 1]
```diff
@@ -3,10 +3,17 @@ module.exports = {
...
-  ]
+  ],
+  'browser_args': {
+      'Chrome': [
+          '--disable-gpu',
+          '--headless',
+          '--remote-debugging-port=9222',
+          '--window-size=1440,900'
+      ],
+  }
```
Comment (Turbo87) [inline blueprints/app/files/testem.js]: indentation here should use 2 spaces, not 4

File: tests/fixtures/smoke-tests/js-testem-config/testem.js [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
+    "launch_in_dev": [
+        "Chrome"
+    ],
+    "browser_args": {
+        "Chrome": [
+            "--disable-gpu",
+            "--headless",
+            "--remote-debugging-port=9222",
+            "--window-size=1440,900"
+        ],
```
Comment (Turbo87) [inline tests/fixtures/smoke-tests/js-testem-config/testem.js]: indentation again

**Comment by sivakumar-kailasam** [comment]
Fixed the indentation.

**Comment by twokul** [comment]
@sivakumar-kailasam maybe the diff is funky but it seems like there are still 4 spaces used for indentation

**Comment by sivakumar-kailasam** [comment]
@twokul developers using two spaces better be paid better compared to those using 4 😅. The whole file wasn't indented properly, which is now fixed.

**Comment by twokul** [comment]
@sivakumar-kailasam just trying to keep things consistent mate

**Comment by twokul** [comment]
@sivakumar-kailasam @Turbo87 seeing failures on appveyor related to installing Google Chrome (manifested in https://github.com/ember-cli/ember-cli/pull/7151). 

```
GoogleChrome v59.0.3071.109 [Approved]
googlechrome package files install completed. Performing other installation steps.
WARNING: Unable to find the architecture of the installed Google Chrome application
File appears to be downloaded already. Verifying with package checksum to determine if it needs to be redownloaded.
Error - hashes do not match. Actual value was 'FD473B8609DF272C1C64150E9C868FF76FBEC13A9FF006E1ADCD915BE23ED295'.
Downloading googlechrome 64 bit
  from 'https://dl.google.com/tag/s/dl/chrome/install/googlechromestandaloneenterprise64.msi'
Progress: 100% - Completed download of C:\Users\appveyor\AppData\Local\Temp\1\chocolatey\GoogleChrome\59.0.3071.109\googlechromestandaloneenterprise64.msi (47.61 MB).
Download of googlechromestandaloneenterprise64.msi (47.61 MB) completed.
Error - hashes do not match. Actual value was 'FD473B8609DF272C1C64150E9C868FF76FBEC13A9FF006E1ADCD915BE23ED295'.
ERROR: Checksum for 'C:\Users\appveyor\AppData\Local\Temp\1\chocolatey\GoogleChrome\59.0.3071.109\googlechromestandaloneenterprise64.msi' did not meet 'e0f1f206a1bb18859019b7f832dea27dcbd22f850b21481a6541a1667a2f9f58' for checksum type 'sha256'. Consider passing the actual checksums through with --checksum --checksum64 once you validate the checksums are appropriate. A less secure option is to pass --ignore-checksums if necessary.
The install of googlechrome was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\GoogleChrome\tools\chocolateyInstall.ps1'.
 See log for details.
Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
Failures
 - googlechrome (exited -1) - Error while running 'C:\ProgramData\chocolatey\lib\GoogleChrome\tools\chocolateyInstall.ps1'.
 See log for details.
Command "choco install googlechrome" failed with exit code -1. Retrying 3 of 3
Chocolatey v0.10.7
Installing the following packages:
googlechrome
By installing you accept licenses for the packages.
```

**Comment by sivakumar-kailasam** [comment]
@twokul addressed in #7152

**Pull request merged**